linux/bmount.go

```diff
@@ -25,9 +25,10 @@ import (
 )
 
 const (
-	procMounts    = "/proc/mounts"
-	mountCommand  = "mount"
-	umountCommand = "umount"
+	procMounts     = "/proc/mounts"
+	mountCommand   = "mount"
+	umountCommand  = "umount"
+	procMountMutex = concurrent.NewMapMutex()
```

Comment:
it should be a variable instead of const.

Reply:
done